### PR TITLE
Update view.feed.php

### PR DESCRIPTION
**Pull request by tonypartridge**
Pull Request for Issue #23542 

### Summary of Changes
We loop on ids but input could be null or empty, make sure it’s array if it is not set.

### CODE REVIEW

**Comment by wlh1950** [comment]
After changing the file I still get:

Invalid argument supplied for foreach() in /joomla/components/com_tags/views/tag/view.feed.php on line 36   (	foreach ($ids as $id))

**Comment by ReLater** [comment]
Try `(array) $app->input->get('id');` instead of `$app->input->get('id', array());`

**Comment by wlh1950** [comment]
(array) $app->input->get('id'); seems to work, no warnings anymore. I'll keep an eye on it.

**Comment by zero-24** [comment]
@tonypartridge can you please apply the requested changes to the PR?

File: components/com_tags/views/tag/view.feed.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function display($tpl = null)
 	{
 		$app       = JFactory::getApplication();
 		$document  = JFactory::getDocument();
-		$ids       = $app->input->get('id');
+		$ids       = $app->input->get('id', array());
```
Comment (Quy) [inline components/com_tags/views/tag/view.feed.php:30]:
```suggestion
		$ids       = (array) $app->input->get('id');
```

**Comment by Quy** [comment]
Closing in favor of PR #24051<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23543">issues.joomla.org/tracker/joomla-cms/23543</a>.</sub>

**Pull request closed without merging**